### DISCUSSION TITLE
Running on Heroku

### DISCUSSION THREAD

**Discussion by cancan101** [geldata/gel#3512] [Ideas]
I was wondering what the best practices are to use edgedb on Heroku. I didn't see any notes [here](https://www.edgedb.com/docs/guides/deployment/index) as to how to do so.

Given my understanding of the edgedb architecture, it seems like the `application -> edgedb -> postgres` in which case edgedb should be runnable as a sort of sidecar / daemon process sitting along side the application and then communicating with a standard (managed) DB service. In that case perhaps using a similar approach to how Heroku suggests integrating pgbouncer with your application which is to use ["Multiple Buildpacks for an App"](https://devcenter.heroku.com/articles/using-multiple-buildpacks-for-an-app) and then run the `edgedb` application as a daemon process in the same dyno as the webserver. I don't know the implications though of having multiple daemons running in different dynos.

**Comment by elprans** [comment]
That's right.  The "sidecar" buildpack approach is one way.  We need to make Heroku buildpack, though, which we haven't yet.
